morgoth/settings.py

```diff
@@ -89,6 +91,13 @@ class Core(Configuration):
         }
     }
 
+    CRONTAB_DJANGO_PROJECT_NAME = 'morgoth'
+
+    CRONJOBS = (
+        ('0 0 * * *', 'django.core.management.call_command', ['update_product_details']),
+        ('0 1 * * *', 'django.core.management.call_command', ['update_addon_groups']),
```

Comment:
In prod, you'll have to be careful to make sure that you have some persistent storage for product details. You won't have that by default.

Another way to deal with this would be to use the DB based storage for product details, which is built in.